src/components/TeacherStatistics.jsx

```diff
@@ -389,27 +393,29 @@ export const CourseDetail = (props) => {
                             <td>{item.course_name}</td>
                             <td>{item.date}</td>
                             <td>{item.booking}</td>
+                            <td>{item.attendances}</td>
                           </tr>
                         </React.Fragment>
                       );
                     })}
                   {!reserved || reserved.length > 0 ? (
                     ""
                   ) : (
-                    <tr>
-                      <td colspan="4">
-                        <div className="w-100 d-flex justify-content-center">
-                          <h5>No Record to show</h5>
-                        </div>
-                      </td>
-                    </tr>
-                  )}
+                      <tr>
+                        <td colspan="4">
+                          <div className="w-100 d-flex justify-content-center">
+                            <h5>No Record to show</h5>
+                          </div>
+                        </td>
+                      </tr>
+                    )}
                 </tbody>
               </Table>
             </div>
           </div>
+          <StaticCharts data={{ bookedLectures, monthLectures }} course_list={myCourses}></StaticCharts>
```

Comment:
`bookedLectures` and `monthLectures` don't contain the statistic for all the courses in `myCourse` but only for those in `courseFilter`. I would suggest to change it
```suggestion
          <StaticCharts data={{ bookedLectures, monthLectures }} course_list={myCourses.filter(c => courseFilter.includes(c.id))}></StaticCharts>
```